<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>380</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>381</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="line_displayFormat">
      <property name="text">
       <string>Display Format</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_changeFormat">
      <property name="text">
       <string>ChangeFormat</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>101</y>
     <width>321</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLineEdit" name="line_time">
      <property name="text">
       <string>Time</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_enterTime">
      <property name="text">
       <string>Time</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>219</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="lbl_displayTime">
      <property name="text">
       <string>Display Time</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_displayTime">
      <property name="text">
       <string>Display Info</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTimeEdit" name="timeEdit_Test">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
